src/main/java/org/fcrepo/migration/foxml11/Foxml11InputStreamFedoraObjectProcessor.java

```diff
@@ -378,6 +378,10 @@ public String getExternalOrRedirectURL() {
             }
         }
 
+        @Override
+        public boolean isFirstVersion() {
+            return getVersionId().split("\\.")[1].equals("0");
```

Comment:
I'm not sure fedora guarantees this naming convention for versions.  It may be that it does.  In any event, if we don't want to depend on that naming convention, we can determine if a version is the first another way... I'll comment on that below.